tests/TestCase/View/ViewVarsTraitTest.php

```diff
@@ -32,7 +33,7 @@ public function setUp()
     {
         parent::setUp();
 
-        $this->subject = $this->getObjectForTrait('Cake\View\ViewVarsTrait');
+        $this->subject = new Controller;
```

Comment:
Because it was telling me $this->request was not being set in the foreach loop of  createView().